### PR TITLE
per http://cernvm.cern.ch/portal/filesystem/cvmfs-2.1.20#the-cvmfs-confi...

### PR DESCRIPTION
**Pull request by jcpunk**
Define the new "cvmfs-config" repo for possible use

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -54,6 +54,7 @@
   $major_release = regsubst($::operatingsystemrelease,'^(\d+)\.\d+$','\1')
   $cvmfs_version          = hiera('cvmfsversion','present')
   $cvmfs_yum              = hiera('cvmfs_yum',"http://cern.ch/cvmrepo/yum/cvmfs/EL/${major_release}/${::architecture}")
+  $cvmfs_yum_config       = hiera('cvmfs_yum',"http://cern.ch/cvmrepo/yum/cvmfs-config/EL/${major_release}/${::architecture}")
```
Comment (traylenator) [inline manifests/params.pp]: This should be just 
```puppet
$cvmfs_yum_config = "http://cern.ch/cvmrepo/yum/cvmfs-config/EL/${major_release}/${::architecture}"
```

as before no new heira calls that I then need to deprecate.  (I'll do that soon.)

File: manifests/yum.pp [hunk 2]
```diff
@@ -41,6 +42,16 @@
     require     => File['/etc/pki/rpm-gpg/RPM-GPG-KEY-CernVM'],
     proxy       => $cvmfs_yum_proxy
   }
+  yumrepo{'cvmfs-config':
+    descr       => "CVMFS config yum repository for el${major}",
+    baseurl     => $cvmfs_yum_config,
+    gpgcheck    => 1,
+    gpgkey      => 'file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CernVM',
+    enabled     => 1,
```
Comment (traylenator) [inline manifests/yum.pp]: There should be a cvmfs_yum_config_enabled variable also I think.

**Comment by traylenator** [comment]
Looks like the repo should be disabled by default as per
http://cvmrepo.web.cern.ch/cvmrepo/yum/cernvm.repo

**Comment by jcpunk** [comment]
squashed, plus no hiera

**Comment by traylenator** [comment]
Hi.
I'm happy to add this and PR #29 then do a release before the deprecations in  PR #34 are added. Please rebase this or PR #29 first, I expect they may conflict so one at a time into master.

**Pull request closed without merging**